### PR TITLE
delete compiles/d

### LINKED ISSUE #103: Remove mentions of "compiled" from docstrings
Since "compiled" is a technical term in the computing context, and the meaning here is more like "combined".

@anikaluo this might be where you got if from

### PR DESCRIPTION
**Pull request by anikaluo**
closes #103

### CODE REVIEW

File: src/metaphlan.jl [hunk 1]
```diff
@@ -84,8 +84,8 @@ end
 """
     metaphlan_profile(path::AbstractString, rank::Union{Int, Symbol}=:all; sample::AbstractString=basename(first(splitext(path))))
 
-Compiles a MetaPhlAn file into a CommunityProfile.
-Can select data according to taxonomic rank. If rank not given, all data is compiled.
+Combines a MetaPhlAn file into a CommunityProfile.
```
Comment (kescobo) [inline src/metaphlan.jl:87]:
```suggestion
Creates a CommunityProfile from a MetaPhlAn output file.
```

File: src/metaphlan.jl [hunk 2]
```diff
@@ -84,8 +84,8 @@ end
...
+Can select data according to taxonomic rank. If rank not given, all data is combined.
```
Comment (kescobo) [inline src/metaphlan.jl:88]:
```suggestion
Can select data according to taxonomic rank. If rank not given, all data is used.
```

File: src/metaphlan.jl [hunk 3]
```diff
@@ -123,8 +123,8 @@ end
 """
     metaphlan_profiles(path::AbstractString, rank::Union{Int, Symbol}=:all; keepunidentified=false)
 
-Compiles MetaPhlAn profiles from a merged table into a CommunityProfile.
-Can select data according to taxonomic rank. If rank not given, all data is compiled.
+Combines MetaPhlAn profiles from a merged table into a CommunityProfile.
```
Comment (kescobo) [inline src/metaphlan.jl:126]:
```suggestion
Reads in MetaPhlAn profiles from a merged table into a CommunityProfile.
```

File: src/metaphlan.jl [hunk 4]
```diff
@@ -123,8 +123,8 @@ end
...
+Can select data according to taxonomic rank. If rank not given, all data is combined.
```
Comment (kescobo) [inline src/metaphlan.jl:127]:
```suggestion
Can select data according to taxonomic rank. If rank not given, all data is used.
```

**Review by kescobo — CHANGES_REQUESTED** [review]

**Comment by anikaluo** [comment]
Yeah, those sentences make much more sense now.

**Review by kescobo — APPROVED** [review]

**Pull request merged**